acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/ibft2/BftMiningAcceptanceTest.java

```diff
@@ -14,20 +14,26 @@
  */
 package org.hyperledger.besu.tests.acceptance.ibft2;
```

Comment:
Thinking this should move into a bft package along ParameterizedBftTestBase and BftAcceptanceTestParameterization